src/xSTIR/pSTIR/STIR.py

```diff
@@ -1193,7 +1201,19 @@ def backward(self, ad, subset_num = 0, num_subsets = 1):
             (self.handle, ad.handle, subset_num, num_subsets)
         check_status(image.handle)
         return image
-
+    def get_linear_acquisition_model(self):
+        am = type(self)()
+        am.set_up( self.acq_templ, self.img_templ )
+        return am
+    def direct(self, image, subset_num = 0, num_subsets = 1, ad = None):
```

Comment:
docstring. what's `ad` anyway?

Reply:
`ad` seems to be `AcquisitionData`, and it is needed by the `forward` method

Reply:
if `ad` is present, the forward projection is put there, otherwise it is returned (so it is like numpy's `out`)

Reply:
can we name it `out`, then? It'll make it much clearer

Reply:
yes please. However, this will need changing `forward` as well. Possibly has consequences for the examples etc. and would be a backwards compatibility break, so please create a separate issue for it, with a separate PR, such that it has some visibility.

Reply:
I used `out` in the `direct` method not in the `forward` so there won't be backward incompatibility.